src/Pos/Launcher/Runner.hs

```diff
@@ -199,6 +203,7 @@ runRealModeDo discoveryCtx slottingCtx transport np@NodeParams {..} sscnp
                runProduction .
                    runResourceT .
                    usingLoggerName lpRunnerTag .
+                   runJsonLogT' mJLHandle .
```

Comment:
@gromakovsky I changed it to `runWithoutJsonLogT` and created a pull request: https://github.com/input-output-hk/cardano-sl/pull/937